cwd/server.js

```diff
@@ -35,5 +35,9 @@ router.use(express.static(__dirname));
 
 router.get('/editor', (req, res) => {
   console.log('Received GET request at /editor');
-  res.sendStatus(200);
+  res.send("okay");
```

Comment:
Any particular reason for doing this instead of 200?

Reply:
Fixed